gemp-lotr/gemp-lotr-async/src/main/web/src/updated/jCard/modules/image-resolution/data/players-committee/2021/april-errata.list.ts

```diff
@@ -0,0 +1,25 @@
+import { Override } from "../../../types/override.interface";
+
+export const aprilErrata: Override[] = [
+    {
+        id: '51_40',
+        url: 'https://i.lotrtcgpc.net/errata/xlist/51_040.jpg'
+    },
+    {
+        id: '51_311',
+        url: 'https://i.lotrtcgpc.net/errata/xlist/51_311.jpg'
+    },
+    {
+        id: '53_42',
+        url: 'https://i.lotrtcgpc.net/errata/xlist/53_042.jpg'
+    },
+];
+
+// {
```

Comment:
these were commented out in the source (still in gemp22/jCard.js)

Reply:
Ah yes.  Those commented entries are cards which have been errata'd more than once (you'll find the key again further down the page), and it was basically just a note indicating where the old URL was at.  There is a (relatively) new naming convention and directory structure for the image files (which kicks in around the "2022 Christmas Errata" section), and I left these in so I didn't miss them while converting (eventually I need to convert them all, I've just been lazy).  

The commented entries here can be omitted from any new setup.

Reply:
thanks for the clarity!

It also brings up a good point that static data additions to the cache startup value should overwrite older values.

So if I update card 1_1 in file A I should be able to in file N add 1_1 again and it should update that cache entry....